testsuite/features/step_definitions/command_steps.rb

```diff
@@ -474,7 +493,13 @@
   node.run('rm /tmp/test_user_defined_state')
 end
 
-Then(/^the cobbler report contains "([^"]*)" for system "([^"]*)"$/) do |arg1, system|
+Then(/^the cobbler report contains "([^"]*)" for system "([^"]*)"$/) do |arg1, host|
```

Comment:
```suggestion
Then(/^the cobbler report contains "([^"]*)" for system "([^"]*)"$/) do |text, system|
```

Reply:
not done. We use "host" when we expect something like 'sle_client' or 'sle_minion'.